packages/jaeger-ui/src/components/TraceDiff/TraceDiffHeader/CohortTable.js

```diff
@@ -124,10 +125,17 @@ export default class CohortTable extends React.PureComponent<Props> {
         <Column
           title="Duration"
           dataIndex="data.duration"
-          key="age"
+          key="duration"
           render={(value, record) => record.state === fetchedState.DONE && formatDuration(value)}
         />
-        <Column title="Spans" dataIndex="data.spans.length" key="address" />
+        <Column title="Spans" dataIndex="data.spans.length" key="spans" />
+        <Column
+          title="Link"
```

Comment:
I don't think we need the "Link" column header... Seems self-explanatory?

Reply:
yeah it looks better without, removed